lisa/platform_.py

```diff
@@ -177,7 +177,7 @@ def prepare_environment(self, environment: Environment) -> Environment:
     def deploy_environment(self, environment: Environment) -> None:
         platform_runbook = cast(schema.Platform, self.runbook)
 
-        log = get_logger(f"deploy[{environment.name}]", parent=self._log)
+        log = get_logger("", parent=environment.log)
```

Comment:
@squirrelsc is this fine?

Reply:
It doesn't need to change the logger prefix. The original prefix can help find log from different sinks.

Reply:
With my change, the log is 
`lisa.env[generated_0]. creating deployment`

Without removing "deploy..."
`lisa.env[generated_0].deploy[generated_0] creating deployment`

With the original platform logger (self._log)
`lisa.[azure].deploy[generated_0] creating deployment`

Reply:
`lisa.env[generated_0].deploy creating deployment` is better.